Common/PkmnModels/Pokeset.cs

```diff
@@ -0,0 +1,18 @@
+using System.Collections.Immutable;
+using System.Runtime.Serialization;
+
+namespace Common.PkmnModels
+{
+    [DataContract]
+    public struct Pokeset
+    {
+        public Pokemon Pokemon { get; set; }
```

Comment:
Will this also have compatibility with pokecat's multiple-options functionality? i.e. lists of abilities/items/etc., and a list of lists of moves

Reply:
no, that is pokecat's concern. now that I think about it calling the other struct "pokeset" might actually be wrong, since in pokecat's terms a pokeset is exactly that, and not just a pokemon with some additional data. The same thing is probably to remove the Pokeset struct from here, since that will probably stay pokecat's concern anyway.

Reply:
yeah, perhaps